### PR TITLE
Add Listening page

### PR DESCRIPTION
**Pull request by ojhasubham**

### CODE REVIEW

File: frontend/src/router.ts [hunk 1]
```diff
@@ -37,6 +37,15 @@ export const router = new Router({
       path: "/sound-editing/:taskId/quality-check/feedback",
       component: () => import("@/views/QCSubmissionForm.vue")
     },
+    {
+      path: "/listen/:fileName",
+      component: () => import("@/views/Listen/ListenLayout.vue"),
```
Comment (anantakrishna) [inline frontend/src/router.ts]: Why not `AnonymousLayout`? I was expecting you to use it.

File: frontend/src/router.ts [hunk 2]
```diff
@@ -37,6 +37,15 @@ export const router = new Router({
...
+      children: [
+        {
+          path: "", component: () => import("@/views/Listen/Listen.vue")
```
Comment (anantakrishna) [inline frontend/src/router.ts]: Can we put this view in the root of the `/views` folder? Now it looks like an artificial folder.

File: frontend/src/views/Listen/Listen.vue [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+<style>
+</style>
+
+<script>
+export default {
+  name: "ListenAudio",
+  data: () => ({
+    fileName: null,
+    storageBaseDomainName: "test.storage.audioseva.purebhakti.info",
```
Comment (anantakrishna) [inline frontend/src/views/Listen/Listen.vue]: No. This should not be hard-coded. Read again https://trello.com/c/59mJXDBs/73-storage-structure.

File: frontend/src/views/Listen/Listen.vue [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    baseUrl: "http://original.",
+    audioUrl: ""
+  }),
+  mounted: function() {
+    this.getAudioUrl();
+  },
+  methods: {
+    getAudioUrl() {
```
Comment (anantakrishna) [inline frontend/src/views/Listen/Listen.vue]: Why not making it a computed property?

File: frontend/src/views/Listen/Listen.vue [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+<style>
+</style>
+
+<script>
+export default {
+  name: "ListenAudio",
+  data: () => ({
+    fileName: null,
+    storageBaseDomainName: "test.storage.audioseva.purebhakti.info",
+    baseUrl: "http://original.",
```
Comment (anantakrishna) [inline frontend/src/views/Listen/Listen.vue]: The name of this property is not correct. You can name it `storageSubdomain` and save there only `original`. But this is not a `baseUrl`. Moreover, you're using it only once, no need making it a property here at all.

File: frontend/src/router.ts (hunk 1 above)
Comment (ojhasubham) [reply frontend/src/router.ts]: I used AnonymousLayout but for the particular listening page, I changed it. But I will use AnonymousLayout.

File: frontend/src/router.ts (hunk 2 above)
Comment (ojhasubham) [reply frontend/src/router.ts]: Sure, I will put it to the root of /view

File: frontend/src/views/Listen/Listen.vue (hunk 3 above)
Comment (ojhasubham) [reply frontend/src/views/Listen/Listen.vue]: ok, I will get it from *process.env.VUE_APP_STORAGE_ROOT_DOMAIN*

File: frontend/src/views/Listen/Listen.vue (hunk 4 above)
Comment (ojhasubham) [reply frontend/src/views/Listen/Listen.vue]: I have made it in computed property

File: frontend/src/views/Listen/Listen.vue (hunk 5 above)
Comment (ojhasubham) [reply frontend/src/views/Listen/Listen.vue]: I have removed this property

File: frontend/src/views/Listen.vue [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+</style>
+
+<script>
+export default {
+  name: "ListenAudio",
+  data: () => ({
+    fileName: null,
+  }),
+  computed: {
+    getAudioUrl: function() {
```
Comment (anantakrishna) [inline frontend/src/views/Listen.vue]: Computed property should not start with `get`. It's just `audioUrl`.

File: frontend/src/views/Listen.vue [hunk 7]
```diff
@@ -0,0 +1,68 @@
+/*
+ * sri sri guru gauranga jayatah
+ */
+<template>
+  <div>
+    <v-toolbar :clipped-left="$vuetify.breakpoint.lgAndUp" fixed app>
+      <v-toolbar-title>
+        <router-link to="/" tag="span" style="cursor: pointer">
+          <h3>Audio File Name: {{ fileName ? fileName.split('.')[0]: null }}</h3>
```
Comment (anantakrishna) [inline frontend/src/views/Listen.vue]: It's not good to put this logic into the template. Rather create another computed property for the base name of the file.

File: frontend/src/views/Listen.vue [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+<script>
+export default {
+  name: "ListenAudio",
+  data: () => ({
+    fileName: null,
+  }),
+  computed: {
+    getAudioUrl: function() {
+      this.fileName = this.$route.params.fileName;
```
Comment (anantakrishna) [inline frontend/src/views/Listen.vue]: I don't think it's a good idea to change the data property inside computed property.

Comment (anantakrishna) [reply frontend/src/views/Listen.vue]: I expect you to propose the most proper way to do it. I'm thinking that the filename can be done a separate computed property.

Comment (ojhasubham) [reply frontend/src/views/Listen.vue]: yes I have Done with separate computed property

**Pull request merged**